vms/platformvm/spend.go

```diff
@@ -354,15 +355,31 @@ func (vm *VM) semanticVerifySpend(
 				err,
 			)
 		}
+
+		// The `semanticVerifySpendUTXOs` called bellow was extended to accept `signers` the owners
+		// that constitute the multisig alias which owns the UTXO being spent.
+		// Returned `inputOwners` slice is type of `secp256k1fx.TransferOutput` which takes part
+		// only in the signature verification in `vm.fx.VerifyTransfer`.
+		inputOwners, err := utxoDB.GetUTXOMultisigOwners(utxo)
+		if err != nil {
```

Comment:
hey Pawel, this block should function for non-multisig utxos also. However, with the current implementation it's presumed that all utxos are multisig and thus an error is returned here although it shouldn't.
I think we need an extra condition here for the 'not found' case. example:
		`if err != nil && err != database.ErrNotFound {`
and an extra check and handling for the database.ErrNotFound

Reply:
Thank you @knikos !
Fixed in [629d614e](https://github.com/chain4travel/caminogo/commit/629d614e8adf5bf3d8c48dff0235c90e5243e11b)